hm_30/hm_030.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import random
+import json
+
+GOOGLE = "https://www.google.com/"
+FACEBOOK = "https://www.facebook.com/"
+TWITTER = "https://twitter.com/"
+AMAZON = "https://www.amazon.com/"
+APPLE = "https://www.apple.com/"
+ALL = [GOOGLE, FACEBOOK, TWITTER, AMAZON, APPLE]
```

Comment:
Since you don't use variables above anywhere else, you can put values directly in the list. No need to store values in separate variables in this case